<!DOCTYPE HTML>
<!--
	Verti by HTML5 UP
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>
	<head>
		<title>Jason Dong - Duke University</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
		<link rel="stylesheet" href="assets/css/main.css" />
	</head>
	<body class="is-preload no-sidebar">
		<div id="page-wrapper">

			<!-- Header -->
				<div id="header-wrapper">
					<header id="header" class="container">

						<!-- Logo -->
							<div id="logo">
								<h1><a href="index.html">JD</a></h1>
							</div>

						<!-- Nav -->
							<nav id="nav">
								<ul>
									<li><a href="index.html">Welcome</a></li>
									<li>
										<li class="current"><a href="index.html#features-wrapper">My Projects</a>
										<ul>
											<li><a href="poker.html">iOS Poker Bankroll Tracker</a></li>
											<li><a href="adsb.html">ADS-B Extension via UAV</a></li>
											<li><a href="cs307.html">Java Board Game Collection</a></li>
											<li><a href="ece350.html">Verilog & MIPS Sludge Runner Game</a></li>
											<li><a href="suntracker.html">Sun Tracking Power Supply</a></li>
											<li><a href="tapebubble.html">Automated Aircraft Damage Inspection and Tape Bubble Study</a></li>
											<li><a href="skunkfeeder.html">Automatic Skunk Feeder</a></li>
											<li><a href="igem2018.html">Engineered Microbes to Sense and Destroy Biofilms</a></li>
											<li><a href="igem2017.html">Engineered Microbes to Sense and Detect ETEC</a></li>

										</ul>
									</li>
									<li><a href="resume.html">Résumé</a></li>
									<li><a href="aboutme.html">About Me</a></li>
									<li><a href="contactme.html">Contact Me</a></li>
								</ul>
							</nav>

					</header>
				</div>

			<!-- Main -->
				<div id="main-wrapper">
					<div class="container">
						<div id="content">

							<!-- Content -->
								<article>

									<h2>Engineered Microbes to Sense and Detect ETEC</h2>
									<h3>MAY 2017 - NOV 2017</h3>

									<center><a href="http://2017.igem.org/Team:US_AFRL_CarrollHS/Mentors" class="button small icon solid fa-arrow-circle-right">Team Website</a></center>
									<br>

									<table>
										<tbody>		
											<tr>
												<td width = "50%">
													<h3>Group Members</h3>	
													<ul>
														<li>Christina Davis</li>
														<li>Dallas McDonald</li>
														<li>Andrea Poole</li>
														<li>Andrea Smith</li>
														<li>Annie Bete</li>
														<li>Jonah Carter</li>
														<li>Jason Dong</li>
														<li>Max Herrmann</li>
														<li>Hayley Jesse</li>
														<li>Peter Menart</li>
													</ul>
												</td>

												<td width="50%">
													<h3>Mentors</h3>
													<ul>
														<li>Dr. Chia Hung</li>
														<li>Dr. Mike Goodson</li>
														<li>Dr. Nancy Kelley-Loughnane</li>
														<li>Dr. Vanessa Varaljay</li>
														<li>Dr. Joseph K. Tang</li>
														<li>Rachel Krabacher</li>
														<li>Dr. Martha Carter</li>
														<li>Dr. Caroline Metosh-Dickey</li>
														<li>Dr. Christina O'Malley</li>
														<li>Dr. Svetlana Harbaugh</li>
														<li>Anna Bennet</li>
													</ul>
												</td>
											</tr>
											<tr>
												<td style="vertical-align: top;">
													<h3>Abstract</h3>
													<p>Each year Enterotoxigenic Escherichia coli (ETEC), a form of pathogenic E. coli known for causing traveler’s stomach, affects millions of travelers. ETEC produces two toxins, a heat labile and a heat stable version. The basis of the project is to complete three different steps: sense ETEC, respond to its presence, and package the sense/response module in a cellulose matrix.</p>

													<p>The goal of the sensing step is to engineer a non-pathogenic E. coli to detect the pathogenic E. coli communication signals. The response section involves engineering the same E. coli to glow green upon the detection of harmful bacteria. In order to package the sense and respond E. coli, cellulose binding domains will be engineered to be expressed on the CsgA portions of curli specific proteins on E. coli.</p>

													<p>We chose this project after brainstorming many problems that United States Air Force personnel face on a daily basis. Traveling for months at a time, those deployed become susceptible to many pathogens in the water supply, especially harmful strains of E. coli. ETEC is the most common cause of traveler's stomach. We can help detect ETEC by using this modular system, potentially eradicating the pathogens from the water and the body.</p>
												<td>
													<br>
													<br>
													<img src = "images/igem-group.jpg" width="100%" alt="RH Team Picture" class="center">
												</td>
											</tr>
											<tr>
												<td>
													<img src = "images/graph.png" width="120%" alt="statistics picture" class="center">
												</td>
												<td style="vertical-align: top;">
													<h3>The Problem</h3>
													<p>There are three clear distinct layers that form in fuel containment systems: A water layer, the sludge layer, and the actual fuel layer. The formation of these layers usually takes a few weeks, but over time, water seeps into these underground tanks due to imperfections with storage techniques. With this water, microorganism contamination also comes into the mix. Now, a perfect environment for these microorganisms is present, because the water provides the perfect living conditions, and the biofuel contains high amounts of FAMEs (Fatty Acid Methyl Esters) for the microorganisms to feed on. Due to the density difference between fuel and water, the fuel sits on top, the water sits on the bottom, and the microorganisms sit in the middle forming the sludge, which is more commonly known as a biofilm.</p>

												</td>
											</tr>	
										</tbody>
									</table>
									<hr>
										<center><h3>Project Description</h3></center>
										<img src = "images/plan.png" width="100%" alt="Plan Overview" class="center">
										<img src = "images/plan2.png" width="100%" alt="Plan Overview 2" class="center">

									<hr>
										<center><h3>Results</h3></center>
										<p>At the conclusion of our project we had created three working plasmids, all of which were sent to the parts registry.</p>
										<p>Part BBa_K2522000 is a plasmid developed using the csgA gene extracted from the genomic DNA of E. coli Nissle, and a double cellulose binding domain taken from the iGEM DNA Distribution Kit. Theoretically, the plasmid should allow an E. coli bacterium to produce curli fibers with a double cellulose binding domain attached to each CsgA subunit so that the bacterium can encase itself in cellulose. However, while sequencing was completed to confirm our plasmid matched our theoretical design, no tests have been performed to validate a working construct yet (Ability to bind to cellulose, ect.</p>
										<p>Part BBa_K2522001 is a working construct developed to produce Super Folder Green Fluorescent Protein in the presence of the quorum sensing molecule 3OC12. The DNA is confirmed to work as theoretically planned through both sequencing and experiment.</p>

										<center><img src = "images/colony.png" width="50%" alt="Picture of flourescent Colonies" class="center"></center>
										<p class="caption" style="margin-left: 30%;margin-right: 30%;">
											<strong>Figure 1:</strong> This Picture shows our plasmid produces a visible flourescense in the presence of 3OC12
										</p>

										<p>Part BBa_K2522002 is a plasmid designed to produce sfGFP in the presence of the quorum sensing molecule Auto Inducer 2 (AI-2). While confirmed to match our theoretical design via sequencing, due to time constraints only one experiment was performed. The plasmid was grown up overnight along with supernatant from a E.coli Nissle 1917 sample (which in theory should contain AI-2); however, the test was inconclusive due to contamination (bacterial growth in pure LB broth used as our blank)</p>	

										<p>We also characterized our two choices for green fluorescent protein: GFPa1 and sfGFP. Below are our results indicating that sfGFP is best for our project due to the short time taken to fluoresce (~ 4 hours to a visible level) and it’s use in a diagnostic environment that our plasmid would be used for.</p>

										<center><img src = "images/plas_gfp.png" width="80%" alt="Picture of flourescent Colonies" class="center"></center>
										<center><img src = "images/plas_gfp2.png" width="80%" alt="Picture of flourescent Colonies" class="center"></center>
										<hr>
										<h3>
											Please check out our website for additional details. I would also like to thank my teammates for helping with the creation of graphics and text.
										</h3>		
								</article>

						</div>
					</div>
				</div>

			<!-- Footer -->
				<div id="footer-wrapper">
					<footer id="footer" class="container">
						<div class="row">
						</div>
						<div class="row">
							<div class="col-12">
								<div id="copyright">
									<ul class="menu">
										<li>&copy; Jason Dong. All rights reserved</li> <li>HTML Template From HTML5UP, Edited For Personal Use</li>
									</ul>
								</div>
							</div>
						</div>
					</footer>
				</div>

			</div>

		<!-- Scripts -->

			<script src="assets/js/jquery.min.js"></script>
			<script src="assets/js/jquery.dropotron.min.js"></script>
			<script src="assets/js/browser.min.js"></script>
			<script src="assets/js/breakpoints.min.js"></script>
			<script src="assets/js/util.js"></script>
			<script src="assets/js/main.js"></script>

	</body>
</html>